On virtual machines, rdrand is already reseeded by the host cpu

no need to waste 10 msec of boot time on this


--- a/arch/x86/kernel/cpu/rdrand.c~	2015-04-12 18:12:50.000000000 -0400
+++ b/arch/x86/kernel/cpu/rdrand.c	2015-04-28 13:54:35.657574408 -0400
@@ -48,6 +48,7 @@
 	if (!cpu_has(c, X86_FEATURE_RDRAND))
 		return;		/* Nothing to do */
 
+	return;
 	for (count = i = 0; i < RESEED_LOOP; i++) {
 		ok = rdrand_long(&tmp);
 		if (ok)
